packages/config/config/devices/0x019b/z-trm6.json

```
@@ -0,0 +1,469 @@
+				},
+				{
+					"label": "33",
+					"value": 4
+				},
+				{
+					"label": "47",
+					"value": 5
+				},
+				{
+					"label": "6,8",
+					"value": 6
+				},
+				{
+					"label": "100",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Internal Sensor Min Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 50,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "5",
+			"label": "Floor Sensor Min Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 50,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "External Sensor Min Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 50,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "7",
+			"label": "Internal Sensor Max Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 400,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "8",
+			"label": "Floor Sensor Max Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 400,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "9",
+			"label": "External Sensor Max Temp Limit",
+			"valueSize": 2,
+			"minValue": 50,
+			"maxValue": 400,
+			"defaultValue": 400,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "10",
+			"label": "Internal Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "11",
+			"label": "Floor Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "12",
+			"label": "External Sensor Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unit": "0.1 °C",
+			"unsigned": false
+		},
+		{
+			"#": "13",
+			"label": "Regulation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Hysteresis",
+					"value": 0
+				},
+				{
+					"label": "PWM",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "14",
+			"label": "Temperature Control Hysteresis",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 30,
+			"defaultValue": 5,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "15",
+			"label": "Temperature Display",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Setpoint",
+					"value": 0
+				},
+				{
+					"label": "Measured",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "16",
+			"label": "Active Display Brightness",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unit": "10%",
```

All units with a number in them should have a space between the number and the actual unit
```suggestion
			"unit": "10 %",
```
please do the same for the other params too.

---

should be fixed for all params this now